tests/overlay.bats

```diff
@@ -29,3 +29,36 @@ load helpers
   run ls ${TESTDIR}/lower/bar
   [ "$status" -ne 0 ]
 }
+
+@test "overlay dest permissions" {
+  if test \! -e /usr/bin/fuse-overlayfs -a "$BUILDAH_ISOLATION" = "rootless"; then
+    skip "BUILDAH_ISOLATION = $BUILDAH_ISOLATION" and no /usr/bin/fuse-overlayfs present
+  elif test "$STORAGE_DRIVER" = "vfs"; then
+    skip "skipping overlay test because \$STORAGE_DRIVER = $STORAGE_DRIVER"
```

Comment:
I don't believe this will ever run in CI, because:   https://github.com/containers/buildah/blob/f3c5c034256bc0d4eb358d8f08439f3a9630f526/tests/helpers.bash#L6

Reply:
Yup that goes for the existing test also.  I am not sure how we fix this.  Is there a way to force these tests to run with overlay?

Anyways I would like to get this merged.